test/standalone/Soil/soiltest.jl

```diff
@@ -147,7 +147,8 @@ end
     )
 
     Y, p, coords = initialize(soil)
-
+    # Here we do not add a dss buffer: check that.
```

Comment:
in this comment do you mean we should add a test that `add_dss_buffer_to_aux` doesn't change `p`?

Reply:
I meant that that is what this next @test is checking. I'll clarify!